mlos_bench/mlos_bench/config/schemas/cli/cli-schema.json

```diff
@@ -62,6 +62,16 @@
             "$ref": "#/$defs/json_config_path"
         },
 
+        "random_init": {
+            "description": "If true, initialize the tunables with random values; otherwise, use the defaults.",
+            "type": "boolean"
+        },
+
+        "random_seed": {
+            "description": "Random seed to use in conjunction with random_init.",
+            "type": "number"
```

Comment:
I think only ints are supported, right?

Reply:
Can also add tests to the "full" cli example to make sure we don't accidentally drop it in the future.

Reply:
done